discovery/gossiper.go

```diff
@@ -399,11 +399,11 @@ type AuthenticatedGossiper struct {
 	// every new block height.
 	blockEpochs *chainntnfs.BlockEpochEvent
 
-	// prematureChannelUpdates is a map of ChannelUpdates we have received
-	// that wasn't associated with any channel we know about.  We store
-	// them temporarily, such that we can reprocess them when a
+	// prematureMessages is a map of ChannelUpdates and NodeAnnouncement we
+	// have received that wasn't associated with any channel we know about.
+	// We store them temporarily, such that we can reprocess them when a
 	// ChannelAnnouncement for the channel is received.
-	prematureChannelUpdates *lru.Cache
+	prematureMessages *lru.Cache
```

Comment:
After this PR lands eventually, then we can also get rid of the current type casting as well: https://github.com/lightninglabs/neutrino/pull/261

Reply:
Dope!